cli/plasmo/src/features/manifest-factory/mv2.ts

```diff
@@ -46,21 +46,27 @@ export class PlasmoExtensionManifestMV2 extends BaseFactory<ExtensionManifestV2>
   }
 
   protected prepareOverrideManifest = () => {
+    const { manifest } = this.packageData
     const output = {
-      ...this.packageData.manifest
-    }
+      ...manifest
+    } as unknown as ExtensionManifestV2
```

Comment:
interesting that you need a double assertion here

Reply:
yeah, it's the case when you need to convert two incompatible type (in this case, we're forcing the package's manifest into the v2 form).